st2common/st2common/util/workflow/mistral.py

```diff
@@ -61,6 +61,11 @@
     }
 }
 
+JINJA_WITH_ST2KV = '{{.*?st2kv\..*?}}'
```

Comment:
Only whitespace can be located before `st2kv\.` string, right?

If so, we should probably make it more restrictive using `\s*` or similar, or am I missing something?